docs/detections/rules-ui-create.asciidoc

```diff
@@ -476,6 +476,10 @@ also affect this rule.
 .. *Building block* (optional): Select to create a building-block rule. By
 default, alerts generated from a building-block rule are not displayed in the
 UI. See <<building-block-rule>> for more information.
+.. **Max alerts per run** (optional): Specify the maximum number of alerts the rule can create each time it runs. Default is 100.
++
+NOTE: Do not set this value higher than the {kibana-ref}/alert-action-settings-kb.html#alert-settings[{kib} configuration setting] `xpack.alerting.rules.run.alerts.max`, which determines the maximum alerts generated by _any_ rule in the {kib} alerting framework.
```

Comment:
From this note, it's unclear why we recommend to not (or even forbid to?) set this value higher than the global setting. Should we explain why we say "do not", or should we just explain what will happen when `max_signals > xpack.alerting.rules.run.alerts.max`? There's nothing dangerous in setting it higher than the global setting, and this note feels too alarming.

Reply:
Suggesting wording along these lines: "When setting the Max alerts per run higher than the {kibana-ref}/alert-action-settings-kb.html#alert-settings[{kib} configuration setting] `xpack.alerting.rules.run.alerts.max`, which determines the maximum alerts generated by _any_ rule in the {kib} alerting framework, the maximum number of alerts created per rule run will be determined by a lower value between rule Max alerts and `xpack.alerting.rules.run.alerts.max`. "

@joepeeples please take a look. cc @banderror

Reply:
Good point @banderror — this was based on the pre-existing note for `max_signals` in [API docs](https://www.elastic.co/guide/en/security/master/rules-api-create.html#opt-fields-all) that warns against this "to avoid rule failures," so we probably should tone that down, too. I can do that in another commit in this PR soon, basically to match whatever we end up here.

@approksiu we might be able to keep most of the original, just adjust the tone and add an example to make the point more concrete:
```suggestion
NOTE: This setting can be superseded by the {kibana-ref}/alert-action-settings-kb.html#alert-settings[{kib} configuration setting] `xpack.alerting.rules.run.alerts.max`, which determines the maximum alerts generated by _any_ rule in the {kib} alerting framework. For example, if `xpack.alerting.rules.run.alerts.max` is set to 1000, the rule can generate no more than 1000 alerts even if **Max alerts per run** is set higher.
```

Reply:
Looks good, thanks @joepeeples !